tests/test_client.py

```diff
@@ -304,6 +309,14 @@ def test_delete_raster():
     client.delete_raster(RASTER_ID)
 
 
+@responses.activate
+def test_delete_detectionarea():
+    RASTER_ID = 'foobar'
+    client = _client()
+    add_mock_delete_detectionarea_response(RASTER_ID)
+    client.remove_raster_detection_areas(RASTER_ID)
+
```

Comment:
Aren't we missing an assert here ? I feel like this tests nothing

Reply:
Yeah we missed it. I also added a commit because I had the same error in other tests too (I wrongly thought that responses will raise an error when an endpoint is not called, instead it does only if we req HTTPO with a not registered url)